### PR TITLE
DHCPServer: config update

### PR DESCRIPTION
**Pull request by HaseenaSainul**

### CODE REVIEW

File: DHCPServer/CMakeLists.txt [hunk 1]
```diff
@@ -27,6 +27,8 @@ set(MODULE_NAME ${NAMESPACE}${PROJECT_NAME})
 
 message("Setup ${MODULE_NAME} v${PROJECT_VERSION}")
 
+set(PLUGIN_DHCPSERVER_AUTOSTART false CACHE STRING true)
```
Comment (artur-gebicz-ml) [inline DHCPServer/CMakeLists.txt:30]: Set is missing the description. It shouldn't be 'true', but something meaningful

**Review by artur-gebicz-ml — CHANGES_REQUESTED** [review]

File: DHCPServer/CMakeLists.txt (hunk 1 above)
Comment (HaseenaSainul) [reply DHCPServer/CMakeLists.txt]: updated

**Review by pwielders — DISMISSED** [review]

**Review by artur-gebicz-ml — CHANGES_REQUESTED** [review]

File: DHCPServer/CMakeLists.txt [hunk 2]
```diff
@@ -27,6 +27,8 @@ set(MODULE_NAME ${NAMESPACE}${PROJECT_NAME})
 
 message("Setup ${MODULE_NAME} v${PROJECT_VERSION}")
 
+set(PLUGIN_DHCPSERVER_AUTOSTART false CACHE STRING "Automatically start DHCPServer plugin")
```
[the remark this answers is not in the document]
Comment (HaseenaSainul) [reply DHCPServer/CMakeLists.txt]: done

File: DHCPServer/CMakeLists.txt [hunk 3]
```diff
@@ -27,6 +27,8 @@ set(MODULE_NAME ${NAMESPACE}${PROJECT_NAME})
...
+set(PLUGIN_DHCPSERVER_OUTOFPROCESS true CACHE BOOL "Controls if the plugin should run in its own process [true, false].")
```
[the remark this answers is not in the document]
Comment (HaseenaSainul) [reply DHCPServer/CMakeLists.txt]: done

**Review by artur-gebicz-ml — APPROVED** [review]

**Pull request merged**